frontend/online-shop/src/services/ProductService.tsx

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+export const getAll = async () => {
+  const result = await axios.get("http://localhost:8080/products", {
```

Comment:
Nice, good use of `axios`. In a real application you would use it the same but instead of sending a body containing the credentials you would send a header containing the JWT/OAUTH/... token. Something like `Authorization: 'Bearer ej8asda...'`